### DISCUSSION TITLE
showing problem in deploying

### DISCUSSION THREAD

**Discussion by chuchu387** [vercel/community#6081] [Help]
### Summary

Your Team encountered an unknown problem is showing while deploying

### Steps to Reproduce

![Screenshot 2024-03-10 172214](https://github.com/vercel/community/assets/84653546/a4f341bf-82b8-428f-8271-05aa7f0eaa9f)

**Comment by amyegan** [comment]
Thanks for reporting this. It was also reported in https://github.com/vercel/community/discussions/6027. Let's move over there to keep the conversation in one place.
